### ISSUE TITLE
Change current location

### ISSUE THREAD

**Issue by Dommfom** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#257]
It would be much better to be able to change your default location to be more exact so that Assistant gives more accurate weather reports.

**Comment by Li-cyber-dot** [comment]
I sees as though I am facing a similar problem- I guess for now the best solution is to manually speak your location to the GA!

**Comment by Dommfom** [comment]
I agree, though would still be great to save on that extra bit of effort!
